packages/generator/src/imports.ts

```diff
@@ -40,7 +40,7 @@ export function externalImportDeclaration(
 
 export function coreImportDeclaration(
   namedImports: string[],
-  oDataVersion: ODataVersion
+  oDataVersion?: ODataVersion
```

Comment:
```suggestion
```

Reply:
This function is exported and therefore this could be a breaking change, right? (Assuming someone uses strict typescript it would no longer compile because it would not allow a function to be called with unnecessary arguments.

Reply:
But it is not exported externally (not from the root index.ts).

Reply:
yes this can go.